CMakeLists.txt

```diff
@@ -126,16 +125,10 @@ list(APPEND _Foundation_common_build_flags
     "-Wno-switch"
     "-fblocks")
 
-if(CMAKE_SYSTEM_NAME STREQUAL "WASI")
-  list(APPEND _Foundation_common_build_flags
-    "-D_WASI_EMULATED_SIGNAL"
```

Comment:
Just confirming - is it intentional that we no longer provide this define to the C targets / we don't provide `HAVE_STLCPY`/`HAVE_STRLCAT` at all?